src/providers/fault-count/fault-count.ts

```diff
@@ -87,6 +88,36 @@ export class FaultCountProvider {
     };
   }
 
+  public getVehicleChecksFaultCountCatC = (vehicleChecks: CatCUniqueTypes.VehicleChecks): VehicleChecksScore => {
```

Comment:
is there a way we can reuse the B+E function as this is an exact duplicate apart from the type. If not then can we have some unit tests for this function please

Reply:
I have not looked into this function right now as Vehicle checks come under another story (struggling to find right now, trying to find the number). I've just renamed the function so that the WRTC page references cat c specifics.

Reply:
MES-4037 is what covers the above. Refactoring will come under that ticket. Right now just needed to remove none CAT-C or common references